pkg/commands/archive.go

```diff
@@ -137,3 +155,17 @@ func unTAR(baseDir string, r io.Reader) error {
 		}
 	}
 }
+
+func isSupportedArchive(header []byte) bool {
+	if detectCompression(header) {
+		return true
+	}
+	r := tar.NewReader(bytes.NewBuffer(header))
```

Comment:
We should because docker does. I will verify. If it doesn't I will add it

Reply:
Actually, since we are only currently support .tar.gz, we should just keep the verification as such.
We can add more archive logic later or somehow unify our logic with docker. I think that's beyond this PR and I will just modify my code to detect gz file instead